lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -488,6 +494,44 @@ newDBLayer trace fp timeInterpreter = do
             deleteWhere [ BlockSlot >. point ]
             -- TODO: remove dangling metadata no longer attached to a pool
 
+        delistPools [] = pure ()
+        delistPools pools =
```

Comment:
This hasn't been tested yet, because we don't have SMASH integration tests.

Reply:
Yes, this is gonna be tested first manually. I got sidetracked of trying to use SMASH as you might know. For now I'll focus on manual tests.